### PR TITLE
Implement feedback

### PR DESCRIPTION
**Pull request by novemberborn**
Based on feedback from https://github.com/sindresorhus/ava/pull/760.

cc @sindresorhus @jamestalmage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,8 +41,31 @@ To get the path to an npm package it's best to use
...
+
+```js
+const hash = sync(require.resolve('babel-core/package.json'), 'seed value')
+```
+
 Currently only a synchronous interface is available.
 
+## API
+
+### `sync(paths[, seed])`
```
Comment (sindresorhus) [inline README.md:63]: Unrelated, but curious, why use `sync(paths[, seed])` over `sync(paths, [seed])`? The latter IMHO looks better.

**Comment by sindresorhus** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (jamestalmage) [inline README.md:63]: That's the way I always used to see it. It groups the comma with the optional value it acts as a separator for. It's how Node does it, and is useful when you want to represent a [more complex argument structure](https://nodejs.org/dist/latest-v5.x/docs/api/child_process.html#child_process_child_send_message_sendhandle_options_callback).

Comparing these two:

``` js
fn1(a[, b[, c]]);

fn2(a[, b][, c]);
```

It's apparent that for `fn1` you must specify a value for `b` if you want to specify a value for `c`. For `fn2`, you can just do `fn2(a, c)` - and the method will use type checking to infer which construct you meant.

Your way allows a bit of ambiguity:

``` js
fn2(a, [b], [c]);
```

You can't tell if arguments are determined strictly on position or if there's some type-checking, arg-shifting magic happening under the covers. Yes, you absolutely could develop a convention, but the slightly less pretty format leaves little doubt.

**Comment by novemberborn** [comment]
Thanks!

**Pull request merged**